rsocket-core/src/main/java/io/rsocket/metadata/AuthMetadataCodec.java

```diff
@@ -37,7 +37,8 @@ public static ByteBuf encodeMetadata(
     }
     if (actualASCIILength < 1 || actualASCIILength > 128) {
       throw new IllegalArgumentException(
-          "custom auth type must have a strictly positive length that fits on 7 unsigned bits, ie 1-128");
+          "custom auth type must have a strictly positive length that fits on 7 unsigned bits, ie"
+              + " 1-128");
```

Comment:
Does not seem that his line was changed
```suggestion
          "custom auth type must have a strictly positive length that fits on 7 unsigned bits, ie 1-128");
```

Reply:
google formatter changed this when I ran it from jar (v 1.9). I used ./gradlew goJF now